documentation/library_guide/parallel_api/range_based_api.rst

```diff
@@ -1,15 +1,107 @@
 Range-Based API Algorithms
 ##########################
-.. Note::
-
-  The use of the range-based API requires C++17 and the C++ standard libraries coming with GCC 8.1 (or higher)
-  or Clang 7 (or higher).
 
 C++20 introduces the Ranges library. C++20 standard splits ranges into two categories: factories and adaptors.
 A range factory does not have underlying data. An element is generated on success by an index or by dereferencing an iterator.
 A range adaptor, from the |onedpl_long| (|onedpl_short|) perspective, is a utility that transforms the base range,
 or another adapted range, into a view with custom behavior.
 
+|onedpl_short| only supports random access ranges, because they allow efficient and constant-time access to elements at any position in the range. This enables effective workload distribution among multiple threads or processing units, which is crucial for achieving high performance in parallel execution.
+
+|onedpl_short| introduces two sets of range based algorithms:
+
+* The oneapi::dpl::ext::ranges namespace supports integration with the Ranges Library coming with C++20 standard and introduced by std::ranges namespace, allowing you to leverage oneDPL parallel algorithms with the standard ranges paradigm. The functionality is implemented for the host and the device execution policies and requires C++20.
+* The oneapi::dpl::experimental::ranges namespace supports integration with oneDPL ranges introducing by oneapi::dpl::experimental::ranges namespace, allowing you to leverage oneDPL parallel algorithms with the range functionality like the Ranges Library from C++20 standard. The functionality is implemented for the device execution policies only and requires C++17.
+
+.. Note::
+
+  The use of the oneapi::dpl::ext::ranges requires C++20 and the C++ standard libraries coming with GCC 10 (or higher) or Clang 10 (or higher).
```

Comment:
Instead of saying "C++ standard libraries coming with GCC 10", can we just directly reference the actual libraries (libstdc++ and libc++), or do you think users may not be as familiar with these names?

Reply:
It's subjective, but I feel in this case it is better to refer to the compiler version.